content/news/2022-01-06-bevy-0.6/index.md

```diff
@@ -0,0 +1,1007 @@
+<div class="release-feature-authors">authors: @guimcaballero</div>
+
+Bevy Transforms now have friendly "directional" functions that return relative vectors:
+
+```rust
+// Points to the left of the transform
+let left: Vec4 = transform.left();
+// Points to the right of the transform
+let right: Vec4 = transform.right();
+// Points up from the transform
+let up: Vec4 = transform.up();
+// Points down from the transform
+let down: Vec4 = transform.down();
+// Points forward from the transform
+let forward: Vec4 = transform.forward();
+// Points back from the transform
+let back: Vec4 = transform.back();
+```
+
+### Transform Builder Methods
+
+<div class="release-feature-authors">authors: @Lythenas</div>
+
+Transforms now have helpful `with_translation()`, `with_rotation()`, and `with_scale()` builder methods:
+
+```rust
+Transform::from_xyz(0.0, 0.0, 10.0).with_scale(Vec3::splat(2.0))
+```
+
+## Rust 2021
+
+<div class="release-feature-authors">authors: @mockersf, @YohDeadfall</div>
+
+Bevy has been updated to use Rust 2021. This means we can take advantage of the new Cargo feature resolver by default (which the new WGPU version requires). Make sure you update your projects to Rust 2021 or you will need to manually enable the new feature resolver!
+
+```toml
+[package]
+name = "your_app"
+version = "0.1.0"
+edition = "2021"
+```
+
+## Input
+
+### Gamepads Resource
+
+<div class="release-feature-authors">authors: @CrazyRoka</div>
+
+**Bevy 0.6** adds a `Gamepads` resource, which automatically maintains a collection of connected gamepads.
+
+```rust
+fn system(gamepads: Res<Gamepads>) {
+    // iterates every active game pad
+    for gamepad in gamepads.iter() {
+    }
+}
+```
+
+### Input "any" variants 
+
+<div class="release-feature-authors">authors: @DJMcNab</div>
+
+`Input` collections now have an `any_pressed()` function, which returns true when any of the given inputs are pressed. 
+
+```rust
+ fn system(input: Res<Input<KeyCode>>) {
+    if input.any_pressed([KeyCode::LShift, KeyCode::RShift]) {
+        println!("one or both of the two shift keys are pressed");
+    }
+ }
+```
+
+### Window Transparency 
+
+<div class="release-feature-authors">authors: @louisgjohnson</div>
+
+Winit's "window transparency" feature is now exposed in Bevy's Window type. This allows users to build "widget like" apps without backgrounds or window decorations (on platforms that support it). Here is a Bevy app with a transparent background, rendering a Bevy Logo sprite on top of my Linux desktop background. Seamless! Cool!
+
+![transparent window](transparent_window.png)
+
+### FromReflect Trait and Derive
+
+<div class="release-feature-authors">authors: @Davier</div>
+
+Types can now derive the new `FromReflect` trait, which enables creating "clones" of a type using arbitrary `Reflect` impls. This is currently used to make reflected collection types (like `Vec`) work propely, but it will also be useful for "round trip" conversions to and from `Reflect` types.
```

Comment:
```suggestion
Types can now derive the new `FromReflect` trait, which enables creating "clones" of a type using arbitrary `Reflect` impls. This is currently used to make reflected collection types (like `Vec`) work properly, but it will also be useful for "round trip" conversions to and from `Reflect` types.
```
Typo with properly.